pango/pango.gemspec

```diff
@@ -39,10 +39,17 @@ Gem::Specification.new do |s|
   ]
   s.files += Dir.glob("lib/**/*.rb")
   s.files += Dir.glob("ext/#{s.name}/*.{c,h,def,rb}")
-  s.files += Dir.glob("test/**/*")
 
   s.add_runtime_dependency("cairo-gobject", "= #{s.version}")
   s.add_runtime_dependency("gobject-introspection", "= #{s.version}")
+  s.required_ruby_version = '>= 2.6'
```

Comment:
Could you revert this?
We can't maintain this. We specify this only when we dropped support for not EOL-ed Ruby. (We'll not do it.)

Reply:
done